### ISSUE TITLE
'lumber update' or 'lumber help update' commands throws exception

### ISSUE THREAD

**Issue by lfcipriani** [ForestAdmin/lumber#158]
Everything was going fine until I updated the data model. When running `lumber update` I got the message:

```
> lumber update                      
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'parse-database-url'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/lfcipriani/.asdf/installs/nodejs/10.7.0/.npm/lib/node_modules/lumber-cli/lumber-update.js:9:20)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:266:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:596:3)
```

I can run the node app but `update` return this error (it happens even when I type "lumber help update"). Tried to reinstall npm packages as well. I don't want to regenerate because I already did changes to models.

It happened with node 10.7.0 and 8.9.1.

I'm using latest version of lumber as well.

Thanks

**Comment by jreg11755** [comment]
still happening to me, already reported in #152

**Comment by arnaudbesnier** [comment]
@lfcipriani, @jreg11755 Shame on us guys... The fix is here: https://github.com/ForestAdmin/lumber/commit/03e28d8178279115cf88f893e16437e110bb84a5

I will release an new version in a few minutes.

**Comment by arnaudbesnier** [comment]
Fix released in the new package version (v1.10.3)
Thanks for the feedback guys!

🌲🌲🌲
